src/components/buttons/link-button/link-button.js

```diff
@@ -50,7 +50,7 @@ const LinkButton = props => (
           size: 'medium',
           theme: props.isDisabled ? 'grey' : 'green',
         })}
-      <Text.Body>{props.label}</Text.Body>
+      <Text.Body isInline>{props.label}</Text.Body>
```

Comment:
Nit: `isInline={true}`. I didn't see us abbreviating that here. If we would maybe renaming to `inline` would be apt but I don't see that breaking change happening.